.misc/deps.sh

```diff
@@ -10,8 +10,12 @@ case $CIRCLE_NODE_INDEX in
 esac
 
 # apt-get commands
+sudo sh -c 'echo "deb http://cran.rstudio.com/bin/linux/ubuntu precise/" >> /etc/apt/sources.list'
+gpg --keyserver keyserver.ubuntu.com --recv-key E084DAB9
+gpg -a --export E084DAB9 | sudo apt-key add -
```

Comment:
Are L13-L15 really needed ? Wouldn't `apt-get install r-base r-base-dev` work ? I don't really think we'd have to use the cran mirrors since `r-base` is a standard package.